week1/Program.cs

```diff
@@ -0,0 +1,89 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "This is the homework for week1!");
+
+//task1
+app.MapGet("/string-manipulation", (string input) => {
+string reversed = ReverseString(input); 
+string ReverseString( string s )
+{
+    char[] charArray = s.ToCharArray();
+    Array.Reverse(charArray);
+    return new string(charArray);
+}
+return $"Reversed input value {input}: {reversed}";
+// For input = "world", response: "dlrow"
+});
+
+//task2
+app.MapGet("/string-math", (string input) => {
+    string sentence=input.ToLower();
+int vowelCount = GetVowelCount(sentence); 
```

Comment:
an option to merge these two lines is to do `int vowelCount = GetVowelCount(input.ToLower());`
- your option is perfectly fine, this is just an alternative 😊